src/graphql/entries.ts

```diff
@@ -1,39 +1,96 @@
 import { ContentTypeProps, EntryProps } from 'contentful-management/types';
 
+import { CollectionItem, SysProps } from '../types';
 import { isPrimitiveField, logUnrecognizedFields } from './utils';
 
 /**
  * Updates GraphQL response data based on CMA entry object
  *
- * @param contentType entity
- * @param data the GraphQL response to be updated
- * @param update CMA entry object containing the update
- * @param locale code
+ * @param contentType ContentTypeProps
+ * @param data Record<string, unknown> - The GraphQL response to be updated
+ * @param update EntryProps - CMA entry object containing the update
+ * @param locale string - Locale code
+ * @returns Record<string, unknown> - Updated GraphQL response data
```

Comment:
```suggestion
 * @returns Updated GraphQL response data
```

The return type shouldn't be correct as it is without the `sys` information